community/consistency-check-legacy/src/main/java/org/neo4j/legacy/consistency/ConsistencyCheckTool.java

```diff
@@ -211,7 +195,12 @@ void exitTool()
                 getCause().printStackTrace( System.err );
             }
 
-            exitHandle.pull();
+            exit();
         }
     }
+
+    private static void exit()
+    {
+        System.exit( 1 );
```

Comment:
Yes we can :smile: , but not as part of this PR, in this way we preserve current behaviour and we have separate card about alignment of our tools (and their exit codes as well) in a backlog. 
Will address your comment as part of that work.